snmp/assets/monitors/traps_linkDown.json

```diff
@@ -2,11 +2,11 @@
   "version": 2,
   "created_at": "2023-06-15",
   "last_updated_at": "2023-06-27",
-  "title": "Interface went down on device {{snmp_device.name}}",
+  "title": "Interface is down on SNMP device",
   "tags": [
     "integration:snmp"
   ],
-  "description": "Notify your team when a linkDown trap is received. You can use this monitor as a template for setting up any traps monitor.",
+  "description": "An SNMP trap is an automatic event dispatched by a device to signal critical events, such as status alterations or system malfunctions. This monitor tracks instances where the interface of any network device within a specified namespace reports a shutdown. You can use this monitor as a template for setting up any traps monitor.",
```

Comment:
**nitpick:** reading through this, it may still be useful which trap is used for this recommended monitor - maybe add the specific trap like the previous mentioned - so something like:

> This monitor tracks instances where the interface of any network device within a specified namespace reports a shutdown via a linkDown trap

Reply:
oops - seeing it got updated, forgot descriptions can't be too long 😅 

maybe something like: (under 300 char)
```An SNMP trap is an event dispatched by a device. This monitor tracks instances where the interface of any network device within a specified namespace reports a shutdown (via linkDown trap). You can use this monitor as a template for setting up any traps monitor.```

but again, very nitpicky, customers may already realize this :) feel free to lmk!

Reply:
Very good point, just updated the desc in b08983d
Thanks a lot!